template/android/app/src/main/res/values/styles.xml

```diff
@@ -4,6 +4,7 @@
     <style name="AppTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
```

Comment:
To keep the same styling, maybe consider making `parent="Theme.ReactNative.AppCompat.Light"` to match what `AndroidManifest.xml` was?